docs/src/man/services.md

```diff
@@ -0,0 +1,147 @@
+| `epost`     | Upload or append a list of UIDs to the Entrez History server.              |
+| `esummary`  | Retrieve document summaries for a list of UIDs.                            |
+| `efetch`    | Retrieve formatted data records for a list of UIDs.                        |
+| `elink`     | Retrieve UIDs linked to an input set of UIDs.                              |
+| `egquery`   | Retrieve the number of available records in all databases by a text query. |
+| `espell`    | Retrieve spelling suggestions.                                             |
+| `ecitmatch` | Retrieve PubMed IDs that correspond to a set of input citation strings.    |
+
+["The Nine E-utilities in
+Brief"](https://www.ncbi.nlm.nih.gov/books/NBK25497/#_chapter2_The_Nine_Eutilities_in_Brief_)
+summarizes all of the server-side programs corresponding to each function.
+
+In this package, queries for databases are controlled by keyword parameters. For
+example, some functions take `db` parameter to specify the target database.
+Functions listed above take these parameters as keyword arguments and return
+a `Response` object as follows:
+```jlcon
+julia> using Bio.Services.EUtils      # import the nine functions above
+
+julia> res = einfo(db="pubmed")       # retrieve statistics of the PubMed database
+Response(200 OK, 18 headers, 27360 bytes in body)
+
+julia> write("pubmed.xml", res.data)  # save retrieved data into a file
+27360
+
+shell> head result.xml
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE eInfoResult PUBLIC "-//NLM//DTD einfo 20130322//EN" "https://eutils.ncbi.nlm.nih.gov/eutils/dtd/20130322/einfo.dtd">
+<eInfoResult>
+        <DbInfo>
+        <DbName>pubmed</DbName>
+        <MenuName>PubMed</MenuName>
+        <Description>PubMed bibliographic record</Description>
+        <DbBuild>Build161024-2207m.1</DbBuild>
+        <Count>26590895</Count>
+        <LastUpdate>2016/10/25 02:06</LastUpdate>
+
+```
+
+Let's see a few more examples of parameters.  The `term` parameter specifies a
+search string (e.g. `esearch(db="gene", term="tumor AND human[ORGN]")`).  The
+`id` parameter specifies a UID (or accession number) or a list of UIDs (e.g.
+`efetch(db="protein", id="NP_000537.3", rettype="fasta")`, `efetch(db="snp",
+id=["rs55863639", "rs587780067"])`). The complete list of parameters can be
+found at ["The E-utilities In-Depth: Parameters, Syntax and
+More"](https://www.ncbi.nlm.nih.gov/books/NBK25499/).
+
+When a request succeeds the response object has a `data` field containing
+formatted data, which can be saved to a file as demonstrated above. However,
+users are often interested in a part of the response data and may want to
+extract some fields in it. In such a case,
+[EzXML.jl](https://github.com/bicycle1885/EzXML.jl) is helpful because it offers
+lots of tools to handle XML documents. The first thing you need to do is
+converting the response data into an XML document by `parsexml`:
+```jlcon
```

Comment:
The example code includes nondeterministic results because the databases of NCBI are continuously updated. So, it would be almost impossible to keep the tests green.